xml_converter/integration_tests/test_cases/xml_category_inheritance/input/pack/xml_file.xml

```diff
@@ -1,7 +1,7 @@
 <OverlayData>
     <MarkerCategory Name="mycategory" XPos="169.81" YPos="210.65" ZPos="215.83"></MarkerCategory>
 
-    <MarkerCategory Name="NestedLevel1" XPos="000" MapID="999">
+    <MarkerCategory Name="NestedLevel1" XPos="000" MapID="999" IconFile="texture_one.png" Texture="texture_one.png" TrailData="trail.trl" AnimationSpeed="1">
```

Comment:
What do you think about spiting the category into two instead of adding arbitrary arguments that correspond to trails vs icons. This way we wont accidentally run into this issue in the future if new required attributes are added to the schema.

To me it seems like this solution as written is a "if we add these irrelevant fields it happens to test the thing we want".

Reply:
I hadn't considered the required attributes. 

My thought in adding them to this category was to ensure that the attributes were sorted correctly into either Icons or Trails.